codalab/apps/web/templates/web/competitions/edit.html

```diff
@@ -246,7 +246,7 @@ <h3 class="panel-title">
                 </li>
             {% endfor %}
         </ul>
-        <button type="button" class="medium blue button add-phase btn btn-secondary pull-right">Add Phase</button>
+{#        <button type="button" class="medium blue button add-phase btn btn-secondary pull-right">Add Phase</button>#}
```

Comment:
What are we doing with this?

Reply:
I think this is a remnant from something else, annotate is saying this was edited in 2018 ... ?

Reply:
this shouldn't be showing up any more?